scss/_detail.scss

```diff
@@ -0,0 +1,61 @@
+%title {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+%content {
+  color: $dark-grey;
+  font-size: 16px;
+}
+
+.detail {
+  display: flex;
+  max-width: 600px;
+  padding: 16px;
+  width: 100%;
+
+  &__cover {
+    display: flex;
+    margin-right: 32px;
+
+    &__img {
+      width: 250px;
```

Comment:
You can set the width like:

```css
max-width: 250px;
width: 100%;
```

That way, the image will occupy the maximum space it has 😄

Reply:
Done 😄